grammatical-parsers/grammatical-parsers.cabal

```diff
@@ -73,6 +73,7 @@ test-suite            doctests
   hs-source-dirs:     test
   default-language:   Haskell2010
   main-is:            Doctest.hs
+  other-modules:      README
```

Comment:
This can't be right. There is no such file as README, and README.md is already specified the way it's supposed to be.

Reply:
If I download the source package from hackage, I get the following error with `cabal new-test`:

```
Test suite doctests: RUNNING...

<no location info>: error: can't find file: test/README.lhs
Test suite doctests: FAIL
```

So there's something wrong with how cabal reads the source file to build the sdist.

Reply:
I see. I hadn't noticed that because I run tests in development with all files present.

I'll have to look for a different way to specify doctests.

Reply:
No problem. It is breaking the build for `grammatical-parsers` in nixpkgs, but if you don't see an easy way to make it right we can easily turn off tests there.

Reply:
I must apologize: change you suggested is perfectly correct and I have just pushed it and published it on Hackage. `stack test` now works properly, and I gave up on `cabal v2-test`.

Reply:
No worries, thanks for fixing.